README.md

```diff
@@ -0,0 +1,64 @@
+# Code Specifications
+
+This code specification is for better personal code habit and better team collaboration.
+
+Read this in other languages: [简体中文](./README.zh-CN.md).
+
+## Features
+
+This code specifications contains following two characters:
+
+- Standard local coding:
+  - We use [Editorconfig](https://editorconfig.org/) for basic format support. Refer to config files [.editorconfig](./.editorconfig) .
+  - We use [Prettier](https://prettier.io/) and VSCode editor setting `editor.formatOnSave` for format code at coding time. Refer to config files [.prettierrc.js](./.prettierrc.js) .
+  - We use [ESLint](https://eslint.org/) to complement Prettier. Refer to config files [.eslintrc.json](./.eslintrc.json) .
+- Staged files check:
+  - We use [husky](https://github.com/typicode/husky) work with Git hook for run some command such as Lint, Test, etc .
+  - We use [lint-staged](https://github.com/okonet/lint-staged) work with need commit files for better performance.
+
+## Prerequisite
+
+As this code specifications strongly depend on code editor [VSCode](https://code.visualstudio.com/) , so we need install it.
+
+Also, we need install several VSCode plugins: [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) , [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) , [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) .
+
+## How to use
+
+We have two kinds of usage for this code specifications :
+
+- For pure usage.
+- Wanna to make contribution.
```

Comment:
I suggest moving everything about contributing to a dedicated section called **Contributing**.

Reply:
Ok, accept your good suggestions !